.github/actions/download_remote_config_backup/action.yml

```diff
@@ -0,0 +1,30 @@
+name: Remote Config Backup Download
+description: Download remote config backup
+inputs:
+  REMOTE_CONFIG_URL:
+    description: "Remote config backup URL"
+    required: true
+  GITHUB_TOKEN:
+    description: "Github token"
+    required: true
+runs:
+  using: composite
+  steps:
+    - name: Download remote config backup
+      run: |
+        echo "Downloading remote config backup"
+        curl ${{ inputs.REMOTE_CONFIG_URL }} > packages/api/features.json
+        echo "Remote Config backup downloaded successfully!"
+      shell: bash
+    - name: commit changes
+      if: github.ref == 'refs/heads/master'
```

Comment:
just in case that the action is triggered by merge to master it will be pushed to the repo.

Reply:
Otherwise it will be just copied to the codebase and will be used by the build step

Reply:
so basically if they enable/disable any options in the config, there should be one commit from our side as well so the new config will be downloaded and updated in our codebase, right?

Reply:
with keeping this in mind that this is just for the case that the remote config server is down, if we want to have it on the codebase then one commit to the master and one build for production is needed.